tests/nightly/nightly-kickoff.sh

```diff
@@ -26,7 +26,9 @@ nightly_list_var="5-1-Distributed-Switch \
 5-11-Multiple-Cluster \
 5-12-Multiple-VLAN \
 5-13-Invalid-ESXi-Install \
-5-14-Remove-Container-OOB"
+5-14-Remove-Container-OOB \
+13-1-vMotion-VCH-Appliance \
```

Comment:
We need these tags to be added to the html templates as well for email reporting. LGTM otherwise.

<tr>
<td>
vMotion-VCH-Appliance:
</td>
<td>
${nightlystatus[13]}
</td>
</tr>